src/index.html

```diff
@@ -7,6 +7,353 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header>
+      <a href="#home">
+        <img
+          src="./images/logo.png"
+          alt="moyo"
+          class="img-logo"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="nav__link is-active">
+            APPLE
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+             SUMSUNG
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+           СМАРТФОНЫ И ТЕЛЕФОНЫ
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#"
+              class="nav__link"
+              data-qa="hover"
+            >
+              НОУТБУКИ И КОМПЬЮТЕРЫ
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+            ГАДЖЕТЫ
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+            ПЛАНШЕТЫ
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+            ФОТО
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+             ВИДЕО
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap" rel="stylesheet">
+
+    <main class="catalog">
+      <div class="catalog-row">
+        <div class="card" data-qa="card">
+          <img src="./images/imac.jpeg"
+            alt="imac"
+            class="card__pic"
+          >
+          <p class="card__name">Моноблок APPLE A1419 iMAC 27" Retina 5K (MNED2UA/A)</p>
+          <p class="card__code">Код товара: 195434</p>
+
+          <div class="card__review">
+            <div class="stars--4 stars">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+
+            <div class="card__review-count">Отзывов: 5</div>
+
+
+          </div>
+
+          <div class="card__price">
+            <div>Цена:</div>
+            <div class="card__price-number">69 999 грн</div>
+          </div>
+
+          <a
+          href="#"
+          class="card__buy"
+          data-qa="hover">
```

Comment:
```suggestion
          data-qa="card-hover">
```
add data-qa="card-hover" (not just hover) to the link Купить inside the first card